src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    if (invertedRobot.hasOwnProperty(robot[key])) {
+      return null;
+    }
+
+    invertedRobot[String(robot[key])] = key;
```

Comment:
The explicit conversion to String is not necessary when setting the property on 'invertedRobot'. Object keys are automatically converted to strings in JavaScript, so you can directly use 'invertedRobot[robot[key]] = key;'.